edge/pkg/metamanager/process.go

```diff
@@ -183,6 +189,13 @@ func (m *metaManager) processUpdate(message model.Message) {
 		beehiveContext.Send(EdgeFunctionModel, message)
 	case EdgeFunctionModel:
 		sendToCloud(&message)
+	case cloudmodules.DeviceControllerModuleName:
+		resp := message.NewRespByMessage(&message, OK)
+		sendToCloud(resp)
+
+		message.SetRoute(modules.MetaGroup, modules.DeviceTwinModuleName)
```

Comment:
why we need update the route info and not use the original?

Reply:
here the metamanager changes the source to meta so that the classifyMsg function of the devicetwin can recgonize the messages from meta and handle them as DMI message.

Reply:
I sugget we do not update the route info, we can update the logic in the classifyMsg